pkg/operator/operator.go

```diff
@@ -52,6 +52,12 @@ type Operator struct {
 	log     logr.Logger
 }
 
+// Aggregate kubebuilder RBAC tags in one location for simplicity.
```

Comment:
I thought that the idea of having these comments in the same file as `Reconcile()` was to keep them closer to where it's more visible which Kubernetes resources are used.

Reply:
I guess this is a subjective thing. IMO it's nicer to have all of the RBAC markers in the same spot so you don't have to bounce between files to find a specific RBAC tag. Thoughts? I don't feel super strongly either way, but I think I would prefer to leave all of the RBAC markers in one place, since this is how the contour operator does it https://github.com/projectcontour/contour-operator/blob/main/internal/operator/operator.go

Reply:
Ok, let's keep it consistent between these 2 operators.